UI/log-redaction.hpp

```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
+#include <regex>
+
+using namespace std;
```

Comment:
I don't think this should be in a header file, it can have unintended side effects.

Reply:
It's been awhile; fixed.